app/views/comment_mailer/_comment_notification_footer_for_other.text.erb

```diff
@@ -7,4 +7,4 @@ Review comments on "<%= @comment.ultimate_parent.commentable_name %>": <%= polym
 Reply to this comment: <%= comment_url(@comment, :add_comment_reply_id => @comment.id, :only_path => false) %><% end %>
 Go to the thread starting from this comment: <%= comment_url(@comment, :only_path => false) %><% unless @comment.id == @comment.thread %>
 Go to the thread to which this comment belongs: <%= comment_url(:id => @comment.thread, :only_path => false) %><% end %>
-Read all comments on "<%= @comment.ultimate_parent.commentable_name %>": <%= polymorphic_url(@comment.ultimate_parent, :view_full_work => true, :show_comments => true, :anchor => :comments, :only_path => false) %>
+<% if @comment.original_ultimate_parent.kind_of?(Chapter) && @comment.ultimate_parent.chaptered? %>Read all comments on Chapter <%= @comment.original_ultimate_parent.position %> of "<%= @comment.ultimate_parent.commentable_name %>": <%= work_chapter_url(@comment.original_ultimate_parent.work, @comment.original_ultimate_parent, show_comments: true, anchor: :comments) %><% else %>Read all comments on "<%= @comment.ultimate_parent.commentable_name %>": <%= polymorphic_url(@comment.ultimate_parent, view_full_work: true, show_comments: true, anchor: :comments, only_path: false) %><% end %>
```

Comment:
You can add some whitespace in the code and then use the `<%-` and `-%>` erb tags, they strip whitespace before/after them.